tests/org.dataflowanalysis.analysis.converter.tests/src/org/dataflowanalysis/analysis/converter/tests/ConverterTests.java

```
@@ -0,0 +1,303 @@
+import org.dataflowanalysis.analysis.converter.*;
+import org.dataflowanalysis.analysis.converter.microsecend.*;
+import org.dataflowanalysis.analysis.converter.webdfd.*;
+import org.dataflowanalysis.analysis.core.*;
+import org.dataflowanalysis.analysis.pcm.PCMDataFlowConfidentialityAnalysisBuilder;
+import org.dataflowanalysis.analysis.pcm.core.seff.*;
+import org.dataflowanalysis.analysis.pcm.core.user.*;
+import org.dataflowanalysis.analysis.testmodels.Activator;
+import org.dataflowanalysis.dfd.datadictionary.Assignment;
+import org.dataflowanalysis.dfd.datadictionary.DataDictionary;
+import org.dataflowanalysis.dfd.datadictionary.Pin;
+import org.dataflowanalysis.dfd.dataflowdiagram.DataFlowDiagram;
+import org.dataflowanalysis.dfd.dataflowdiagram.Flow;
+import org.dataflowanalysis.dfd.dataflowdiagram.Node;
+import org.dataflowanalysis.dfd.dataflowdiagram.dataflowdiagramPackage;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.emf.ecore.resource.ResourceSet;
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+import org.eclipse.emf.ecore.xmi.impl.XMIResourceFactoryImpl;
+
+public class ConverterTests {
+	private static final String packagePath="src/org/dataflowanalysis/analysis/converter/tests/";
+	
+	@Test
+	@DisplayName("Test Web -> DFD -> Web")
+	public void webToDfdToWeb() {
+		Main.readWeb(packagePath+"minimal.json");
+		Main.readDFD(packagePath+"minimal",packagePath+"test.json");
+		
+		ObjectMapper objectMapper = new ObjectMapper();        
+		File file = new File(packagePath+"minimal.json");
+		DFD webBefore = null;
+        try {
+            webBefore = objectMapper.readValue(file, DFD.class);
+        } 
+        catch (IOException e) {}
+        
+        objectMapper = new ObjectMapper();        
+		file = new File(packagePath+"test.json");
+		DFD webAfter = null;
+        try {
+            webAfter = objectMapper.readValue(file, DFD.class);
```

Tests should not silently catch exceptions but fail